ts/updater/updater.ts

```diff
@@ -117,15 +117,6 @@ export async function checkForUpdates(
       `[updater] checkForUpdates updateVersionFromFsFromRenderer ${updateVersionFromFsFromRenderer} releaseChannelFromFsFromRenderer ${releaseChannelFromFsFromRenderer}`
     );
 
-    if (releaseChannelFromFsFromRenderer !== 'latest') {
-      // we only allow pre-release updates if the release channel is alpha
-      autoUpdater.allowPrerelease = releaseChannelFromFsFromRenderer === 'alpha';
-      autoUpdater.allowDowngrade = releaseChannelFromFsFromRenderer === 'alpha';
-      logger.info(
-        `[updater] checkForUpdates we are on the ${releaseChannelFromFsFromRenderer} channel allowPrerelease ${autoUpdater.allowPrerelease} allowDowngrade ${autoUpdater.allowDowngrade}`
-      );
-    }
```

Comment:
I feel like the log line with the if could still be useful. What do you think?

```ts
if (releaseChannelFromFsFromRenderer !== 'latest') {
      logger.info(
        `[updater] checkForUpdates we are on the ${releaseChannelFromFsFromRenderer} channel allowPrerelease ${autoUpdater.allowPrerelease} allowDowngrade ${autoUpdater.allowDowngrade}`
      );
    }```

Reply:
Fixed